src/Components/Tip/Tip.jsx

```diff
@@ -1,16 +1,21 @@
-import React from 'react';
-import './Tip.css';
-
-function Tip({ setBillData, tipAmountPerPerson, totalAmountPerPerson }) {
+import React from "react";
+import "./Tip.css";
 
+function Tip({
+  setBillData,
+  tipAmountPerPerson,
+  totalAmountPerPerson,
+  resetCustomTip,
+}) {
   const resetForm = (event) => {
-    event.preventDefault()
+    event.preventDefault();
     setBillData({
       bill: "",
       tipPercent: "",
-      numOfPeople: ""
-    })
-  }
+      numOfPeople: "",
+    });
+    resetCustomTip("");
```

Comment:
This is very logical to me (calling resetCustomTip inside of resetForm). Nice work!